### ISSUE TITLE
Help Aloushka with installation issue

### ISSUE THREAD

**Issue by stefmolin** [stefmolin/Hands-On-Data-Analysis-with-Pandas-2nd-edition#40]
*Migrating this from a comment on a commit to an issue.*

### Required attestation
- [ ] I have **completely and exactly** followed the virtual environment setup instructions from the book and have consulted the [README](https://github.com/stefmolin/Hands-On-Data-Analysis-with-Pandas-2nd-edition#notes-on-environment-setup) for any updates.
- [ ] I have cloned either this repository or my fork of this repository so that I have all necessary files on my local machine.
- [ ] I have checked that this issue has not already been reported or resolved [here](https://github.com/stefmolin/Hands-On-Data-Analysis-with-Pandas-2nd-edition/issues?q=is%3Aissue+label%3Atroubleshooting).
- [ ] I am aware that there is a [pre-built Binder environment](https://mybinder.org/v2/gh/stefmolin/binder-environments/2nd_edition?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgithub.com%252Fstefmolin%252FHands-On-Data-Analysis-with-Pandas-2nd-edition%26urlpath%3Dlab%252Ftree%252FHands-On-Data-Analysis-with-Pandas-2nd-edition%252F%26branch%3Dmaster) that I can use, but I want to install locally on my machine instead.

---

##### 1. Which OS are you using?
Windows

##### 2. Which Python version are you using?
3.11

##### 3. Are you using `conda` or `venv`?
`venv`

---

### Commands run and their outputs
Please provide **all** of the commands you ran as well as the traceback:

<details>

```
TODO: paste commands and any traceback here
```

</details>

### Screenshots
Optionally, include any screenshots that will help diagnose the issue.
![image](https://user-images.githubusercontent.com/24376333/224502170-766d286e-cfd1-4617-8d16-87aa6af07f61.png)

---

Hello Everyone,
I am relatively new to coding , have some basic knowledge and I am trying to apply the knowledge in the book Hands on Data Analysis with Pandas.

I am having an absolute nightmare with setting up the virtual environment as per the instructions of the book. If i automatically load the libraries as per the requirements.txt text file, it comes back with errors.
So i decided to manually load them up individually, one by one, bt the problem is some libraries versions are not accepted (old versions ???) , and i tried instead to use the latest ones. but still, when i run the file "checking up your set up,", i get a mish mash of results. ( see attached snapshot) Some load okay, some are failing because of the different versions used.
Is it okay to use a different more updated version, or will that throw errors when the program is run. Any idea how to get the latest requirements text file?

In Summary, is there a better way to load these libraries without errors?

With Gratitude,

@Aloushka

**Comment by stefmolin** [comment]
@Aloushka - your Python version is too high (see the first failure in the screenshot). You are using 3.11, but only 3.7, 3.8, and 3.9 are supported. If you install different versions than what are in the requirements, you will have problems later.

Since you are new to Python, I would recommend you use the [pre-built Binder environment](https://mybinder.org/v2/gh/stefmolin/Hands-On-Data-Analysis-with-Pandas-2nd-edition/master?urlpath=lab) so you don't have to worry about installation until you are more comfortable. Once you have some experience, you should try using `conda` as discussed in the book since that will also fix the Python version for you.

**Comment by Aloushka** [comment]
Hey Stef.. 

Just wanted to say thank you so much for your help in providing a solution. I am using binder now, and it all works great 

Again, my deepest gratitude 

Aloushka
